aim/tools/cli/commands/manager.py

```diff
@@ -240,6 +264,8 @@ def load_domains(ctx, replace, enforce):
             res = resource.PhysicalDomain(name=phys, monitored=True)
             print_resource(manager.create(aim_ctx, res, overwrite=True))
 
+        if mappings:
+            do_mappings(aim_ctx, manager, replace)
```

Comment:
I don't think that --replace with load_domains should carry over to do_mappings - we probably don't want to accidentally erase any manually created host-domain mapping entries when user updates the config file domain information.

Reply:
I think there's still a use case for blowing away the old mappings. I could make this a separate parameter?

Reply:
Blowing away old mappings can be done through the new command (load-mappings). Do you need to duplicate that there?

Reply:
Sounds good. Will change.